tests/area/test_area.py

```diff
@@ -190,6 +191,20 @@ def test_tick(mock_match_recommendations, config):
         area.tick()
         assert manager.mock_calls == [call.match(), call.update_matcher()]
 
+    @staticmethod
+    @patch("gsy_e.models.market.forward.ConstSettings.ForwardMarketSettings."
+           "ENABLE_FORWARD_MARKETS", True)
+    def test_forward_market_mapping_for_matcher():
```

Comment:
Nit: unit tests usually target one specific public method. That's why we usually use names like `test_<method-name>` or similar for tests. Because of this, I tried to understand what this method is testing, and I have the feeling that this test is conflating tests for `Area._update_myco_matcher()` and `MycoMatcher.update_area_uuid_forward_markets_mapping()` together. In this case I don't think it's a big deal, but I'd suggest you to check ["The magic tricks of testing" by Sandi Metz](https://youtu.be/URSWYvyc42M) and the unit-testing workshop slides (that are just a short version of her talk).